docs/HACKING.md

```diff
@@ -1,6 +1,29 @@
 ⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
 
 # Hacking on the WMCO
+Requires all pre-requisites in the [README](/README.md) to be fufilled.
+
+## User Workflows
+
+### I want to install an officially released version of WMCO on my OpenShift/OKD cluster
+* Instructions to install the community operator can be found in the [README](/README.md).
+  Paying customers should instead look at the [OpenShift docs.](https://docs.openshift.com/container-platform/latest/windows_containers/enabling-windows-container-workloads.html)
+
+### I want to try a specific commit of WMCO
+* [Build the operator](#build)
+* [Deploy the operator directly](#deploying-the-operator-directly)
+
+### I want to try a specific commit of WMCO, and deploy it using OLM
+* [Build the operator](#build)
+* [Deploy the operator through OLM](#deploying-the-operator-through-olm)
+
+### I want to deploy a specific commit of WMCO, in a way that is as close to the official deployment method as possible
```

Comment:
`I want to try the unreleased WMCO, and deploy it mimicking the official deployment method`